modules/SampleModule/inc/samplemodule.hpp

```diff
@@ -4,28 +4,51 @@
 #include "module.hpp"
 #include "server.hpp"
 
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+#pragma once
+
+// Module specific publisher/subscriber
+namespace ROSChannels {
+    namespace SmOnInitResponse {
+        const std::string topicName = "smOnInitResponse";
+        using message_type = std_msgs::msg::Empty;
+		const rclcpp::QoS defaultQoS = rclcpp::QoS(rclcpp::KeepAll());
+        class Pub : public BasePub<message_type> {
+        public:
+            Pub(rclcpp::Node& node, const rclcpp::QoS& qos = defaultQoS) : BasePub<message_type>(node, topicName, qos) {}
+        };
+
+		class Sub : public BaseSub<message_type> {
+		public:
+			Sub(rclcpp::Node& node, std::function<void(const message_type::SharedPtr)> callback, const rclcpp::QoS& qos = defaultQoS) : BaseSub<message_type>(node, topicName, callback, qos) {}
+		};
+    }
+}
+
```

Comment:
I think this should be with the rest of the publishers/subscribers under `common` -> `roschannels`, to have everything in the same place and to avoid confusion in the future.

Reply:
I am hesitant to move it there though since SampleModule is not meant to be run together with other nodes. 
But I agree for every other circumstance.

Reply:
creating a header called testing_channels.hpp or similar might work?

Reply:
Yeah that could work. If have moved it to test_channels.hpp in common.